app/ts/stores.ts

```diff
@@ -28,7 +34,7 @@ function fetchPayloadFromStorage() {
 	// @TODO: Change this to track minimum amount of ETH needed to deposit
```

Comment:
make this an issue

Reply:
https://github.com/DarkFlorist/bouquet/issues/41